### PR TITLE
Delegation Flow Docs Enhancements including Change Your Delegation to a new Operator

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/delegate-to-an-operator.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+title: Delegate to an Operator
+---
+
 # Delegate to an Operator
 
-Follow these steps to initiate delegation of your restaked balance to an Operator. Delegation is an all or nothing operation. You must delegate all of your available Restaked balance to a single Operator.
+Follow the steps below to initiate delegation of your restaked balance to an Operator. Delegation is an all or nothing operation. You must delegate all of your available Restaked balance to a single Operator.
 
-**Step 1:** Navigate to the Operator tab to view a list of available Operators.
+**Step 1:** Navigate to the **Operator** page to view a list of available Operators.
```
Comment (dabit3) [inline docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/delegate-to-an-operator.md:10]: can / should we link to this page?

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/redelegation-process.md [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+---
 
-Redelegation is available for multiple scenarios for both LST and Native Restakers:
-- Restaker would like to undelegate from their current operator, restake, and then delegate to a new operator.
-- A user who accidentally queues a withdrawal, but would like to resume restaking and delegation without having to exit and re-enter their validators from the beacon chain.
+# Change Your Delegation to a New Operator
 
-Please see the following section [Undelegate from an Operator and Initiate Withdrawal](./undelegate-from-an-operator-and-initiate-withdrawal.md)
\ No newline at end of file
+The following use case is intended for a Restaker that would like to change their Delegated balance to a New Operator.
```
Comment (dabit3) [inline docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/redelegation-process.md:8]: totally nitpicking here, but Delegate seems to have some instances uppercase and some instances lowercase without much consistency

**Comment by wesfloyd** [comment]
Closing this PR and moving it to https://github.com/Layr-Labs/eigenlayer-docs/pull/175 to avoid merge conflicts.

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/delegate-to-an-operator.md (hunk 1 above)
Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/delegate-to-an-operator.md:10]: @dabit3  I can definitely see the benefits in terms of usability of linking to that page. If we link to the page here, should we include links in the other page references we make in the docs?

File: docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/redelegation-process.md (hunk 2 above)
Comment (wesfloyd) [reply docs/eigenlayer/restaking-guides/0-restaking-user-guide/stage-2-testnet/testnet-restaker-delegation/redelegation-process.md:8]: @dabit3  I agree completely, maybe we should come up with some rules?

For now, my intention has been:
- Lower case for verbs like "restake", "delegate(d)", etc.
- Upper case for personas like "Restaker", "Operator".
- Upper case for actions like "click the Delegate button".

But if you have seen a better practice, let's adopt it.

**Pull request closed without merging**